Header header
int32[] identificador
float32[] pan
float32[] tilt

